server.js

```diff
@@ -1,20 +1,125 @@
 const express = require("express");
 const cors = require("cors");
+const uuid = require("uuid");
+const moment = require("moment");
+const validator = require("email-validator");
 
 const app = express();
 
 app.use(express.json());
+app.use(express.urlencoded({ extended: true}));
 app.use(cors());
 
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
 
+
+function isInvalidId(id, index, response) {
+  if (index < 0) {
```

Comment:
It is not the only way an ID can be invalid. There is a chance that it won't be a number.